crates/sui-framework/sources/governance/sui_system.move

```diff
@@ -234,14 +234,14 @@ module sui::sui_system {
         self: &mut SuiSystemState,
         delegation: &mut Delegation,
         staked_sui: &mut StakedSui,
-        withdraw_amount: u64,
```

Comment:
Changing the argument name here and other places to make it more clear that the amount is in pool tokens.